tracer.go

```diff
@@ -426,6 +426,14 @@ func newTracer(opts TracerOptions) *Tracer {
 	t.setLocalInstrumentationConfig(envSanitizeFieldNames, func(cfg *instrumentationConfigValues) {
 		cfg.sanitizedFieldNames = opts.sanitizedFieldNames
 	})
+	if apmlog.DefaultLogger != nil {
+		defaultLogLevel := apmlog.DefaultLogger.Level()
```

Comment:
Hasn't this already potentially changed from the local ENV to the remote config? I don't see how this falls back to the original value in case a remote config value was provided but isn't any longer.

Reply:
Not 100% sure I follow your question.
- `apmlog.DefaultLogger.Level()` is called when constructing the Tracer, so there's no remote config yet received. We're just capturing the initial (default/environment variable) value.
- the function literal passed into `t.setLocalInstrumentationConfig` below will be invoked immediately (having no effect, since it's just setting the level to the same value), and also when remote config is set and later removed. The latter bit is taken care of by https://github.com/elastic/apm-agent-go/blob/879c34a347e6486e7cd1767b9e6e3659e23283b6/config.go#L407-L417

Reply:
I missed `defaultLogLevel := apmlog.DefaultLogger.Level()`
in https://github.com/elastic/apm-agent-go/pull/859/files#diff-4e84387cbc4fa01c5f9e572c02575e6c8536608159891072d2fbfa69af83a86fR430 - all good then, thanks.